src/controllers/scripting/controllerscriptenginebase.h

```diff
@@ -25,7 +25,7 @@ class ControllerScriptEngineBase : public QObject {
 
     virtual bool initialize();
 
-    bool executeFunction(QJSValue functionObject, const QJSValueList& arguments = {});
+    bool executeFunction(QJSValue* pFunctionObject, const QJSValueList& arguments = {});
```

Comment:
why the change to pointer? Don't see how that was necessary.

Reply:
Because clang-tidy did not like the use of the copy by value. Maybe some interns have changed in Qt6 that makes the copy expensive?

Reply:
Pretty sure its still cheap. Maybe a clang-tidy bug? From https://doc.qt.io/qt-6/qjsvalue.html#QJSValue-1:

> Note that if other is an object (i.e., isObject() would return true), then only a reference to the underlying object is copied into the new script value (i.e., the object itself is not copied).

Reply:
In any case passing the pointer is one of the cheapest option and it is IMHO better than maintaining a clang-tidy exception.